libs/ui/atoms/src/lib/Icons/__stories__/Icons.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import { storiesOf } from '@storybook/react-native';
+import React from 'react';
+import { Icons } from '../Icons';
+import { NativeBaseProvider } from 'native-base';
+
+const iconsArray = [
+  'add',
+  'arrow-back',
+  'arrow-forward',
+  'arrow-up',
+  'arrow-down',
+  'check',
+  'check-circle',
+  'chevron-down',
+  'chevron-left',
+  'chevron-right',
+  'chevron-up',
+  'circle',
+  'close',
+  'small-close',
+  'menu',
+  'info',
+  'info-outline',
+  'minus',
+  'moon',
+  'question',
+  'question-outline',
+  'search',
+  'sun',
+  'warning-1',
+  'warning-2',
+  'warning-outline',
+  'three-dots',
+  'share',
+  'play',
+  'favourite',
+  'delete',
+  'star',
+  'star-outline',
+  'home',
+  'home-outline',
+  'favorite',
+  'favorite-outline',
+  'plus-circle-outline',
```

Comment:
Maybe it would be better to export this icon Array and use it on Icons.spec.tsx?